odk2-src/app-designer-prereqs.rst

```diff
@@ -116,6 +121,10 @@ If the above command is unsuccessful, some machines may need to append :command:
 
 Should display the installed version of :program:`grunt`. For example the version might be ``grunt-cli v1.2.0``
 
+.. warning::
+
+  If :program:`grunt` is not found, then you need to add it to the *PATH* variable of your system.
```

Comment:
I would modify this slightly to say "then you may need to add it". I agree that this is a likely problem, but I wouldn't want to declare that this is the definite solution.